src/main/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElement.java

```diff
@@ -358,9 +359,27 @@ public void scroll(Location loc) throws CandybeanException {
 	 * element, so links will be activated, radio buttons will be selected, drop boxes will be
 	 * opened, etc.  Use of this method on any non-textfield element is not currently supported.
 	 */
-	public void selectAll() {
-		we.click();
-		we.sendKeys(Keys.END,Keys.chord(Keys.SHIFT,Keys.HOME));
+	public void selectAll() throws CandybeanException {
+		executeJavascript("arguments[0].select()");

```

Comment:
Hey @jmittert , just want to learn about how this works. I know that `executeJavascript` executes in the context of current frame or window. What is `arguments[0]` as you are not passing any arguments in this `executeJavascript`? Is there an assumption before calling `selectionAll()` such as clicking in the textfield before calling this method?